cfgmgr/buffermgrd.cpp

```diff
@@ -109,6 +114,15 @@ shared_ptr<vector<KeyOpFieldsValuesTuple>> load_json(string file)
     }
 }
 
+#if defined(ASAN_ENABLED)
+void sigterm_handler(int signo)
```

Comment:
>sigterm_handler [](http://example.com/codeflow?start=5&length=15)

This function is implemented everywhere. Could you extract it into one single cpp file, and add it into each target in Makefile.

Reply:
@qiluo-msft done